tests/wallet.rs

```diff
@@ -24,6 +24,43 @@ use utils::rand;
 const VALID_TIMEOUT: Duration = Duration::from_secs(5);
 const INVALID_TIMEOUT: Duration = Duration::from_micros(1);
 
+
+
+mod wallet_config {
+    use super::*;
+    
+    #[inline]
+    pub fn new() -> String {
+        json!({
+            "id": rand::random_string(20)
+        }).to_string()
+    }
+
+    #[inline]
+    pub fn with_storage(storage: &str) -> String {
+        json!({
+            "id": rand::random_string(20),
+            "storage_type": storage,
+        }).to_string()
+    }
+
+    #[inline]
+    pub fn with_custom_path<P: AsRef<Path>>(path: P) -> String {
+        json!({
+            "id": rand::random_string(20),
+            "storage_type": "default",
+            "storage_config": {
+                "path": path.as_ref().to_str()
+            }
+        }).to_string()
+    }
+}
+
+#[cfg(test)]
+mod wallet_register {
+
+}
+
```

Comment:
if this is an empty mod, we should just remove it.  if its empty for future work, then its fine to stay....but lets add "test" to the mod name like the others.